ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/tenant_evpn_vxlan/l3leaf-logic.j2

```diff
@@ -59,6 +59,25 @@
 {%         endfor %}
 {%     endif %}
 {% endfor %}
+{# Setting tenant VRF L3 interfaces #}
+{% set leaf.l3_interfaces = [] %}
+{% for tenant in tenants | arista.avd.natural_sort if tenant in leaf.filter_tenants or "all" in leaf.filter_tenants %}
+{%     if tenants[tenant].vrfs is arista.avd.defined %}
+{%         for vrf in tenants[tenant].vrfs if vrf in leaf.vrfs %}
+{%             if tenants[tenant].vrfs[vrf].l3_interfaces is arista.avd.defined %}
+{%                 for entry in tenants[tenant].vrfs[vrf].l3_interfaces %}
+{%                     if entry.nodes is arista.avd.defined and entry.ip_addresses is arista.avd.defined and entry.interfaces is arista.avd.defined and inventory_hostname in entry.nodes %}
+{%                         set node_index = entry.nodes.index(inventory_hostname) %}
+{%                         do entry.update( {'vrf':vrf} )%}
+{%                         do entry.update( {'l3_interface':entry.interfaces[node_index]} )%}
+{%                         do entry.update( {'ip_address':entry.ip_addresses[node_index]})%}
```

Comment:
Really minor but some inconsistent spacing after closing "}" and between closing ")" and "%}".

Reply:
Approved.